clmul/Cargo.toml

```diff
@@ -17,10 +17,6 @@ rand.workspace = true
 rand_core.workspace = true
 criterion.workspace = true
 
-[features]
```

Comment:
There are some refs to `armv8` and `force-soft` dangling in the code and comments. 
Can we also document the new `clmul_armv8` and `clmul_force_soft` flags? We can document them here as a comment.